src/utils/git.rs

```diff
@@ -0,0 +1,10 @@
+use std::process::Command;
+
+pub fn init() {
+    let output = Command::new("sh")
+                     .arg("-c")
+                     .arg("git init")
+                     .output()
+                     .unwrap_or_else(|e| panic!("failed to execute process: {}", e));
```

Comment:
We're not going to shell out to git except for in the git subcommand.  Pull in the `git2` library as a dependency to do this.